f2/src/utils/selfHarmWordsScan.js

```diff
@@ -47,24 +45,36 @@ const selfHarmWordsScan = (data) => {
 //Scan String for key words. Tokenize and stem to identify root words.
 const scanString = (str) => {
   try {
-    let modifiedStr = unidecode(str.toLowerCase())
+    let modifiedStr = str
```

Comment:
Why declare a variable that has the same value as the param?

Reply:
Keep the original copy of data, since modifiedStr will be heavily changed or regrouped in the following code

Reply:
I can consolidate these 2 variables, any others?